bot/exts/info/doc/_cog.py

```diff
@@ -340,11 +340,13 @@ async def get_command(self, ctx: commands.Context, *, symbol_name: Optional[str]
 
             if doc_embed is None:
                 error_message = await send_denial(ctx, "No documentation found for the requested symbol.")
+
```

Comment:
For the sake of the diffs, can you remove this empty linee you added?

Reply:
> For the sake of the diffs, can you remove this empty line you added?

Yep.